### PR TITLE
Status icons

### PR DESCRIPTION
**Pull request by TatianaPan**
I added coloured icons to show book status and improved a bit styling for `Users` table

Screenshot 2020-05-19 at 10 01 56

Screenshot 2020-05-19 at 10 02 30

Screenshot 2020-05-19 at 10 01 39

Screenshot 2020-05-19 at 10 08 04

### CODE REVIEW

File: app/helpers/book_helper.rb [hunk 1]
```diff
@@ -6,4 +6,14 @@ def reservation_button_label(book)
   def next_reservation_status(book)
     book.status == 'available' ? 'reserved' : 'available'
   end
+
+  def status_icon(book)
+    if book.available?
+      image_pack_tag('media/images/icons8-green-circle-50.png', style: 'width: 20px;')
```
[the remark this answers is not in the document]
Comment (ahorincar) [reply app/helpers/book_helper.rb:12]: You could then do something like:
```ruby
icon_path = if book.available? 
                       AVAILABLE_ICON_PATH
                     elsif book.reserved?
                       RESERVED_ICON_PATH
                     elsif book.borrowed?
                       BORROWED_ICON_PATH
                     end

return if icon_path.nil?

image_pack_tag(icon_path, style: 'width: 20px;')
```

File: app/javascript/stylesheets/_sidebar.scss [hunk 2]
```diff
@@ -11,3 +11,11 @@
 #sidebar-wrapper .list-group {
   width: 15rem;
 }
+
+#sidebar-wrapper .sidebar-footer {
+  padding: 0.875rem 1.25rem;
+  font-size: 0.8rem;
+  position: fixed;
```
Comment (ahorincar) [inline app/javascript/stylesheets/_sidebar.scss:18]: I think bootstrap has pseudo classes for these things e.g. `position: fixed`
Would it also be possible to use one of the predefined padding values from Bootstrap? https://getbootstrap.com/docs/4.0/utilities/spacing/

Same with font-size.

**Comment by ahorincar** [comment]
Also, just a suggestion and feel free to ignore it if you thing it's gonna look worse: what do you think about displaying the status icon in front of the book status (e.g. <red icon here> borrowed). That way, the icons are always aligned in the table (whereas now they are not, because the statuses have different lengths)

File: app/helpers/book_helper.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (TatianaPan) [reply app/helpers/book_helper.rb:12]: that looks cleaner, thanks!

**Comment by TatianaPan** [comment]
@ahorincar  thanks a lot for review!
regarding [license:](https://icons8.com/license) seems like link to their page is enough.

> what do you think about displaying the status icon in front of the book status (e.g. < red icon here > borrowed). That way, the icons are always aligned in the table (whereas now they are not, because the statuses have different lengths)

that's a good idea, thank you! I also don't like that icons are not aligned.. 
I think it looks better now.

Screenshot 2020-05-20 at 09 43 31

File: app/javascript/stylesheets/_sidebar.scss (hunk 2 above)
Comment (TatianaPan) [reply app/javascript/stylesheets/_sidebar.scss]: I added pseudo classes for `padding` and `position-fixed`, but haven't found the same for font-size.. As I can see, in Zollinger project you explicitly defined font-size in `.scss` files

Comment (ahorincar) [reply app/javascript/stylesheets/_sidebar.scss]: I'm just curious, do you need to change the font size for the books table only? Or would it be okay to customise the font size of the `<p></p>` tags in general (and therefore have it consistent throughout the app)?

**Pull request merged**